spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaAutoConfiguration.java

```diff
@@ -113,6 +118,20 @@ public KafkaAutoConfiguration(KafkaProperties properties) {
 		return new KafkaTransactionManager<>(producerFactory);
 	}
 
+	@Bean
+	@ConditionalOnProperty(name = "spring.kafka.retry-topic.enabled")
+	@ConditionalOnMissingBean
+	public RetryTopicConfiguration kafkaRetryTopicConfiguration(KafkaOperations<Object, Object> kafkaOperations) {
+		RetryTopicConfigurationBuilder builder = RetryTopicConfigurationBuilder.newInstance();
+		KafkaProperties.RetryTopic retryTopic = this.properties.getRetryTopic();
+		JavaUtils.INSTANCE.acceptIfNotNull(retryTopic.getAttempts(), builder::maxAttempts).acceptIfNotNull(
```

Comment:
I doubt this util from spring-kafka is acceptable in Spring Boot code.
See other similar auto-configuration classes which use something like this: `PropertyMapper map = PropertyMapper.get().alwaysApplyingWhenNonNull();`
The `IntegrationAutoConfiguration` is good one to look into.

Reply:
The `PropertyMapper` class is great, thanks!